cidc_api/app.py

```diff
@@ -32,31 +32,28 @@
 # Check that its auth configuration is validate
 validate_api_auth(app)
 
-# JSON-ify HTTP errors
-@app.errorhandler(HTTPException)
-def jsonify_http_errors(e: HTTPException):
-    """Return JSON instead of default HTML for HTTP errors."""
-    data = {"code": e.code}
-    if hasattr(e, "exc") and isinstance(e.exc, ValidationError):
-        data["message"] = e.data["messages"]
+
+@app.errorhandler(Exception)
```

Comment:
Not sure but maybe BaseException is the proper one?

Reply:
Flask only allows user-defined error handlers for subclasses of `Exception`.